scipy/linalg/flapack.pyf.src

```diff
@@ -735,11 +763,296 @@ interface
           :: lwork=2*minmn+MAX(maxmn,nrhs)
           ! check(lwork>=2*minmn+MAX(maxmn,nrhs))
```

Comment:
Personally I would prefer if we can avoid that LAPACK calls xerbla because the error message is written to stdout. It is better to just get a Python exception which contains useful information.